src/styles/containerComposeStyles/types.ts

```diff
@@ -0,0 +1,6 @@
+import {ViewStyle} from 'react-native';
+import {CSSProperties} from 'react';
+
+type ContainerComposeStyles = Array<ViewStyle | CSSProperties>;
```

Comment:
When we actually import `containerComposeStyles` and use it, will React Native component give an error since it won't accept `CSSProperties`?

Reply:
I checked and turns out you are right, it will give an error 😄 I updated the PR!

Reply:
thank you for checking this ❤️